timemachine/fe/single_topology.py

```diff
@@ -1615,6 +1471,219 @@ def _setup_intermediate_chiral_bond_term(
+            induced_bond_idxs.add(canonicalize_bond((c, i)))
+            induced_bond_idxs.add(canonicalize_bond((c, j)))
+            induced_bond_idxs.add(canonicalize_bond((c, k)))
+        return idxs in induced_bond_idxs
+
+    def _bond_idxs_belong_to_chiral_volume_turning_off(self, idxs):
+        src_chiral_idxs = set(tuple(x) for x in self.src_system.chiral_atom.potential.idxs)
+        dst_chiral_idxs = set(tuple(x) for x in self.dst_system.chiral_atom.potential.idxs)
+        induced_bond_idxs = set()
+        for c, i, j, k in src_chiral_idxs.difference(dst_chiral_idxs):
+            induced_bond_idxs.add(canonicalize_bond((c, i)))
+            induced_bond_idxs.add(canonicalize_bond((c, j)))
+            induced_bond_idxs.add(canonicalize_bond((c, k)))
+        return idxs in induced_bond_idxs
+
+    def _angle_idxs_belong_to_chiral_volume_turning_on(self, idxs):
+        src_chiral_idxs = set(tuple(x) for x in self.src_system.chiral_atom.potential.idxs)
+        dst_chiral_idxs = set(tuple(x) for x in self.dst_system.chiral_atom.potential.idxs)
+        induced_angle_idxs = set()
+        for c, i, j, k in dst_chiral_idxs.difference(src_chiral_idxs):
+            induced_angle_idxs.add(canonicalize_bond((i, c, j)))
+            induced_angle_idxs.add(canonicalize_bond((i, c, k)))
+            induced_angle_idxs.add(canonicalize_bond((j, c, k)))
+
+        return idxs in induced_angle_idxs
+
+    def _angle_idxs_belong_to_chiral_volume_turning_off(self, idxs):
+        src_chiral_idxs = set(tuple(x) for x in self.src_system.chiral_atom.potential.idxs)
+        dst_chiral_idxs = set(tuple(x) for x in self.dst_system.chiral_atom.potential.idxs)
+        induced_angle_idxs = set()
+        for c, i, j, k in src_chiral_idxs.difference(dst_chiral_idxs):
+            induced_angle_idxs.add(canonicalize_bond((i, c, j)))
+            induced_angle_idxs.add(canonicalize_bond((i, c, k)))
+            induced_angle_idxs.add(canonicalize_bond((j, c, k)))
+
+        return idxs in induced_angle_idxs
+
+    def _interpolate_bond(self, idxs, src_params, dst_params, lamb):
+        if self.all_idxs_belong_to_core(idxs):
+            min_max = CORE_BOND_MIN_MAX
+        elif self.any_idxs_belong_to_dummy_a(idxs):
+            if self._bond_idxs_belong_to_chiral_volume_turning_on(idxs):
+                assert 0
+            elif self._bond_idxs_belong_to_chiral_volume_turning_off(idxs):
+                min_max = DUMMY_A_CHIRAL_BOND_CONVERTING_OFF_MIN_MAX
+            else:
+                min_max = DUMMY_A_BOND_MIN_MAX
+        elif self.any_idxs_belong_to_dummy_b(idxs):
+            if self._bond_idxs_belong_to_chiral_volume_turning_on(idxs):
+                min_max = DUMMY_B_CHIRAL_BOND_CONVERTING_ON_MIN_MAX
+            elif self._bond_idxs_belong_to_chiral_volume_turning_off(idxs):
+                assert 0
+            else:
+                min_max = DUMMY_B_BOND_MIN_MAX
+        else:
+            assert 0
+
+        k_min = 0.025
```

Comment:
Add a comment (why 0.025?)

Reply:
(Direct copy-pasta from before) - I don't remember why actually, I think @mcwitt  did the initial tuning for k_min.

Reply:
I'm not sure where that `k_min` came from. It looks like it was added in https://github.com/mcwitt/timemachine/commit/b03f1bcee97e972e0fac4714337482aef4b0bf62.

Reply:
I don't remember either

Reply:
ugh good call - I actually screwed up the k_min here, should be 0.1 instead. I think net effect is that we were ramping up the bonds slower than it should've been doing (k_min=0.025 vs what should be k_min=0.1).

https://github.com/proteneer/timemachine/blob/master/timemachine/fe/single_topology.py#L1657